lib/ruboty/slack_socket_mode/interactive_message.rb

```diff
@@ -0,0 +1,103 @@
+require 'ruboty/message'
+
+module Ruboty
+  module SlackSocketMode
+    class InteractiveMessage
+      attr_reader :robot, :action_id, :payload
+
+      def initialize(robot, payload)
+        @robot = robot
+        @payload = payload
+        @action_id = payload['actions'].first['action_id']
```

Comment:
必ず値が取れるなら、以下のように書けます。
値が取れない可能性があるなら、そのハンドリングが必要だと思います。

```suggestion
        @action_id = payload.dig('actions', 0, 'action_id')
```

Reply:
こちらは BlockKit のボタン、あるいは入力フォームの値を弄った際の payload ですので必ず値が取れる前提でコードを書いている部分になります。頂いたコードを引用させていただきます・・！